airflow-core/src/airflow/ui/src/i18n/locales/hi/common.json

```
@@ -0,0 +1,122 @@
+{
+  "admin": {
+    "Config": "कॉन्फ़िग",
+    "Connections": "कनेक्शन",
+    "Plugins": "प्लग-इन",
+    "Pools": "पूल",
+    "Providers": "प्रदाता",
+    "Variables": "वेरिएबल्स"
+  },
+  "assetEvent_one": "एसेट इवेंट",
+  "assetEvent_other": "एसेट इवेंट्स",
+  "browse": {
+    "auditLog": "ऑडिट लॉग",
+    "xcoms": "एक्सकॉम्स"
+  },
+  "dag_one": "डैग",
```

Is this okay in terms of translation, or is it preferred to keep these terms in English? Or a mix of both, like…

```suggestion
  "dag_one": "डैग (Dag)",
```